oe_paragraphs.install

```diff
@@ -237,3 +237,16 @@ function oe_paragraphs_update_8003(&$sandbox) {
   }
   $field->save();
 }
+
+/**
+ * Fix description for limit field.
+ */
+function oe_paragraphs_update_8004(array &$sandbox): void {
+  if (!$fields = \Drupal::entityTypeManager()->getStorage('field_config')->loadByProperties(['field_name' => 'field_oe_limit'])) {
+    return;
```

Comment:
We can return an error message if field is not found, as we do in the previous update hook:
```
    return t('Could not load default field for content row paragraph.');
```